app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts

```diff
@@ -38,12 +38,6 @@ describe("addDaysToDate", () => {
   });
 });
 
-describe("addDaysToDate", () => {
-  it("should add days to a date and return a date string", () => {
-    expect(addDaysToDate(FAKE_DATE, 2)).toEqual("Mon Dec 14 2020");
```

Comment:
Note that this is the `FAKE_DATE` from in the earlier in the PR diff (although this comment equally applies to L37 above). Note that Dec 12, 2020 was a Saturday, so Saturday + 2 days is Monday. Thursday + 2 days is only a Monday if we don't count weekends. 🤷